specs/credentialmanagement/index.src.html

```diff
@@ -821,9 +867,10 @@ <h4 id="interfaces-request-options">
 
     <pre>
       navigator.<a idl>credentials</a>.<a idl lt="get()">get</a>({
-        "<a idl>types</a>": [ "federated" ],
-        "<a idl>options</a>": {
-            "<a idl>providers</a>": [ "https://identity.example.com/" ]
+        "<a idl>filters</a>": {
+            "federated" : {
+                "<a idl>providers</a>": [ "https://identity.example.com/" ]
```

Comment:
Abstractly, if I have a function that queries a set of things and is expected return one of those things and I want to pass options to that function then an `options` object seems like the right pattern.

If that set contains things of various types and the query options must be type-specific then that options object needs a way of defining that. The only other approach I can think of is something like this (kind of a combination of our approaches) where anything at the top-level is a global option unless the property name is a type name, then it is a type-specific option:

```javascript
navigator.collection.query({
  "types" : ["thingType1", "thingType2", "thingType4"], //Filter the types I want
  "thingType1" : { "thingType1Option1" : true, "thingType1Option2" : 123 }, //Type-specific options for thingType1
  //No options needed for thingType2
  "thingType4" : ["a", "b", "f"], //Type-specific options for thingType1
  "globalOption1" : true,
  "gloablOption2" : function ( arg) { ... }
});
```